### ISSUE TITLE
uninstalled seastar.pc points to wrong paths (to installed location)

### ISSUE THREAD

**Issue by nyh** [scylladb/seastar#566]
This is a spinoff from issue #565.
When I run
```
./configure.py
ninja -C build/release
```

When I try to compile using the **uninstalled** seastar.pc, it doesn't work:
```
$ c++ getting-started.cc `pkg-config --cflags --libs $SEASTAR/build/release/seastar.pc`
c++: error: /usr/local/lib64/libseastar.a: No such file or directory
```
This is because the uninstalled seastar.pc already points to /usr/local/lib64. This is wrong - the uninstalled file should point to the uninstalled libraries, and only during installation it should be changed to point to the installed files.
Moreover, considering that we allow to build different build types (build/release, build/debug, etc.) and each has a different seastar.pc, they should also have different content pointing to the different (uninstalled) libraries. It doesn't make sense for all of them to point to the one which happens to be installed in /usr/local.

**Comment by nyh** [comment]
CC @hakuch

**Comment by nyh** [comment]
This was fixed by commit f2ab53ffebd5f5f1004e3ee7dd3055bb03aa5d58
